profiles/desc/cpu_flags_arm.desc

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2017 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+aes - Use the AES cryptography instruction set
+crc32 - Use the CRC32 instruction set
+iwmmxt - Use the iwMMXt instruction set
+neon - Use the NEON (ASIMD) instruction set
+sha1 - Use the SHA-1 cryptography instruction set
+sha2 - Use the SHA-2 cryptography instruction set
+thumb2 - Enable Thumb-2 instruction set (ARMv6T2 and ARMv7+)
+v4 - Use ARMv4 instructions
+v4t - Use ARMv4T instructions
+v5t - Use ARMv5T instructions
```

Comment:
With a thumb useflag you could drop the t variants:
https://www.altera.com/en_US/pdfs/literature/third-party/archives/ddi0100e_arm_arm.pdf